packages/components/pie-button/package.json

```diff
@@ -1,13 +1,15 @@
 {
   "name": "@justeattakeaway/pie-button",
+  "customElements": "custom-elements.json",
   "version": "0.10.2",
   "description": "PIE design system button built using web components",
   "type": "module",
   "main": "dist/index.js",
   "module": "dist/index.js",
-  "types": "dist/types/index.d.ts",
   "scripts": {
-    "build": "run -T vite build",
+    "build": "yarn create-react-wrapper && run -T vite build && yarn remove-wrapper",
+    "create-react-wrapper": "cd ../../tools/pie-wrappers-react && yarn build:wrapper",
+    "remove-wrapper": "cd ../../tools/pie-wrappers-react && yarn remove:wrapper",
```

Comment:
presumably, we'll be doing this for every web component. Is there any way to prevent adding these commands to every package.json file we create in future?